### ISSUE TITLE
Rule0075 Diagnostic not raised

### ISSUE THREAD

**Issue by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#895]
While resolving https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/890, I've noticed that the example below, the Rule0075 doesn't raise a diagnostic.

```AL
codeunit 50100 MyCodeunit
{
    procedure MyProcedure(var SalesLine: Record "Sales Line")
    var
        SalesInvoiceHeader: Record "Sales Invoice Header";
    begin
        SalesInvoiceHeader.Get(SalesLine."Document Type");
    end;
}
```
